eng/doc/fips/README.md

```diff
@@ -10,40 +10,72 @@ The Go `crypto` package is not FIPS certified, and the Go team has stated that i
 
-These changes are maintained in the `microsoft/dev.boringcrypto*` branches in this repository.
+On Linux, the fork uses [OpenSSL](https://www.openssl.org/) through the [go-crypto-openssl](https://github.com/microsoft/go-crypto-openssl) module. On Windows, [CNG](https://docs.microsoft.com/en-us/windows/win32/seccng/about-cng), using [go-crypto-winnative](https://github.com/microsoft/go-crypto-winnative). Similar to BoringSSL, certain OpenSSL and CNG versions are FIPS 140-2 certified.
 
 It is important to note that an application built with Microsoft's Go toolchain and running in FIPS compatible mode is not FIPS compliant _per-se_. It is on the application development team to use FIPS-compliant crypto primitives and workflows. The crypto runtime will fall back to Go standard library crypto in case it cannot provide a FIPS-compliant implementation, e.g. when hashing a message using `crypto/md5` hashes or when using an AES-GCM cipher with a non-standard nonce size.
 
-## Usage
+## Usage: Build
+
+### Go 1.18
+
+In Go 1.18 and earlier, the Microsoft Go FIPS-compatible builds are are maintained in the `microsoft/dev.boringcrypto*` branches. Only a Linux implementation using OpenSSL is supported for these versions.
+
+1. Get a `1.18-fips` build of the Microsoft Go toolset. See [the microsoft/go readme](https://github.com/microsoft/go#binary-distribution) for options.
+1. Build your Go program using this Go toolset.
+1. The built program now includes the ability to use OpenSSL crypto and FIPS compatibility mode at runtime.
+
+The whole OpenSSL functionality can be disabled even while using a `1.18-fips` Go toolset by building your program with `-tags gocrypt`.
+
+### Go 1.19+
 
-FIPS compatibility mode, and therefore the OpenSSL crypto backend, can be enabled using any of these options:
+In Go 1.19 onward, the FIPS-related changes are maintained in the `microsoft/release-branch.go*` branches in this repository.
+
+1. Get the standard Microsoft build of Go. There is no separate `-fips` build. See [the microsoft/go readme](https://github.com/microsoft/go#binary-distribution) for options.
+1. Enable the desired GOEXPERIMENT and build your program.
+    * To build for Linux/OpenSSL, for example:
+      ```sh
+      GOEXPERIMENT=opensslcrypto go build ./myapp
+      ```
+    * To build for Windows/CNG in PowerShell, for example:
+      ```pwsh
+      $env:GOEXPERIMENT = "opensslcrypto"
+      go build ./myapp
+      ```
+1. The built program will use the specified platform-provided cryptographic library whenever it calls a Go standard library crypto API, and FIPS compatibility can be enabled at runtime.
+
+## Usage: Runtime
+
+To run your program in FIPS compatibility mode, and therefore use the OpenSSL or CNG crypto backend, use any of these options:
```

Comment:
On windows, `GOFIPS=1` does not try to enable FIPS mode because it is not a per-process setting. I would mention that `GOFIPS=1` will crash the process if system-wide FIPS is not enabled.